src/main/webapp/js/user-page-loader.js

```diff
@@ -39,6 +39,9 @@ function showMessageFormIfLoggedIn() {
       })
       .then((loginStatus) => {
         if (loginStatus.isLoggedIn) {
+            if (loginStatus.username == parameterUsername){
```

Comment:
I'm not sure why the about me page is under an `if (loginStatus.username == parameterUsername)` statement. Shouldn't a user be able to see the about me page of others too?

Reply:
Does this mean that you just have to be logged in to view any user's about me page?

Reply:
@emilytran99 I think that's guarded by the `if (loginStatus.isLoggedIn)` statement. Josh can correct me though.

Reply:
I believe what this does is only allow the user to have the ability to make the about me post request if viewing their own account, so they can only update their own about me sections.

Reply:
Got it! Thanks for the clarification!